src/utils.ts

```diff
@@ -9,6 +9,11 @@ import {
 import { formatDate } from './localization';
 
 export const lastUpdated = (data: LeaderboardEntry[]) => {
+  // If we didn't receive any data from the API, set last updated to the current time
```

Comment:
feels like this should be handled in the caller to not display a date, but i do understand a default for empty makes the code cleaner so we can keep